src/test/scala/be/doeraene/sjsreflect/ReflectTest.scala

```diff
@@ -65,6 +71,29 @@ class ReflectTest {
   }
 
   @Test
+  def enumerateClasses(): Unit = {
+    assertTrue(
+      Reflect.enumerateClasses.contains(classOf[EnumeratedClass])
+    )
```

Comment:
To follow the coding style, either write:
```scala
assertTrue(Reflect.enumerateClasses.contains(classOf[EnumeratedClass]))
```
if it fits within 80 characters, otherwise the following:
```scala
assertTrue(
    Reflect.enumerateClasses.contains(classOf[EnumeratedClass]))
```